metadata-ingestion/tests/integration/lookml/lookml_mces_api_bigquery.json

```diff
@@ -1546,20 +1684,6 @@
                         "removed": false
                     }
                 },
-                {
-                    "com.linkedin.pegasus2avro.dataset.UpstreamLineage": {
-                        "upstreams": [
-                            {
-                                "auditStamp": {
-                                    "time": 1586847600000,
-                                    "actor": "urn:li:corpuser:datahub"
-                                },
-                                "dataset": "urn:li:dataset:(urn:li:dataPlatform:bigquery,project-foo.default-db.order,PROD)",
```

Comment:
what happened to this lineage?

Reply:
These was disappear because of condition tag (`{% condition order_region %} order.region {% endcondition %}`) in `liquid.view.lkml`, It is not getting parsed by sqlgalot parser. I added `liquid_variable` in existing configuration so that this tag resolved while resolving liquid templates. Now the lineage appeared in golden file